metricbeat/beater/event.go

```diff
@@ -22,12 +22,21 @@ type eventBuilder struct {
 	host          string
 	startTime     time.Time
 	fetchDuration time.Duration
+	id            eventId
 	event         common.MapStr
 	fetchErr      error
 	filters       *filter.FilterList
 	metadata      common.EventMetadata
 }
 
+// eventId is a unique id for each event or event set.
+type eventId string
+
+func NewEventId() eventId {
```

Comment:
Made it private. As soon as we have the implementation from elasticsearch, we should move it into common.